frontend/src/utils/object/object.ts

```
@@ -0,0 +1,13 @@
+export const isObject = (data: unknown) => !!(data && typeof data === 'object');
+
+export const isObjectAndNotBlank = (data: unknown) =>
+  data && typeof data === 'object' && Object.keys(data).length !== 0;
+
+export const isBlankObject = (data: unknown) =>
+  data && typeof data === 'object' && Object.keys(data).length == 0;
+
+export const hasOwnProperty = (data: unknown, property: string) =>
```

Biome

[error] Do not shadow the global "hasOwnProperty" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)